### PR TITLE
Articles archive page

### PR DESCRIPTION
**Pull request by nikomakela**
Articles archive CMS page.

image

Depends on https://github.com/City-of-Helsinki/react-helsinki-headless-cms/pull/37, but the change is already published in the new HCRC-package.

The route: `/articles` which is also localized to `/artikkelit` (e.g. https://harrastus-helsinki-39.test.kuva.hel.ninja/fi/artikkelit)

### CODE REVIEW

**Comment by nikomakela** [comment]
There are some questions that still needs answers: 

- Should the archive page be visible in the menu? It's not in the designs.
- Should the results be filtered with a selected language? If so, it should be noted that some articles might not have version for every language.
- What information is wanted to be shown in the large card and in the small card?
- Should the large card be changed to a a small card after a search is triggered?

Also, some improvements could be done

- [ ] add some tests
- [ ] the page could be found by using the pageByTemplate -query. Now the URI is hard coded.
- [ ] the page load (from CMS) could be done with an Apollo-component that could be exported from the HCRC. The HCRC -lib does not contain such a component yet.

**Comment by nikomakela** [comment]
Also fixed the article's too big hero image 
image

**Comment by nikomakela** [comment]
>  the page could be found by using the pageByTemplate -query. Now the URI is hard coded.
this does not make much sense if we have have a hardcoded route rewrite for it (for localization reasons)

File: src/constants.ts [hunk 1]
```diff
@@ -3,6 +3,7 @@ import { Language } from './types';
 export const ROUTES = {
   SEARCH: '/search',
   COURSES: '/courses/[eventId]',
+  ARTICLEARCHIVE: '/article-archive',
```
Comment (melniiv) [inline src/constants.ts:6]: ARTICLE_ARCHIVE to be consistent

**Review by melniiv — APPROVED** [review]
Approve with few comments

**Pull request merged**